esterni/manuale-utente/components/supporto-tecnico.tex

```diff
@@ -0,0 +1,32 @@
+\documentclass[../manuale-utente.tex]{subfiles}
+
+\begin{document}
+
+Nel caso venissero riscontrati malfunzionamenti in Stalker, si prega di inviare una mail al seguente \href{mailto:gruppone.swe@gmail.com}{indirizzo di posta}.\\ 
+La mail deve essere preferibilmente nel seguente formato:
+\begin{description}
+    \item[Oggetto:] [Stalker-\textit{Tipo applicazione}] \textit{Nome del malfunzionamento}
+    \begin{itemize}
+        \item \textit{Tipo applicazione} indica la sigla dell'applicazione in cui avviene il malfunzionamento:
+        \begin{description}
+            \item[WA] acronimo di Web Application.
+            \item[MA] acronimo di Mobile Application.
+        \end{description}
+        \item \textit{Nome del malfunzionamento} un testo di massimo 5 parole.
+    \end{itemize}
+    
+    \item[Corpo:] vanno inserite le seguenti informazioni:
+    \begin{itemize}
+        \item data in formato YYYY-MM-DD, e possibilmente ora, in cui è stato riscontrato il problema.
```

Comment:
Mi sembra troppo restrittivo dire ad un utente anche in che formato deve essere la data

Reply:
A me invece sembra ragionevole. E' un modo per disciplinare un'utente nel caso ci voglia contattare.